README.md

```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
 [![Build Status](https://travis-ci.org/Asymmetrik/node-fhir-server-core.svg?branch=develop)](https://travis-ci.org/Asymmetrik/node-fhir-server-core) [![Known Vulnerabilities](https://snyk.io/test/github/asymmetrik/node-fhir-server-core/badge.svg?targetFile=package.json)](https://snyk.io/test/github/asymmetrik/node-fhir-server-core?targetFile=package.json)
 
 
-The Asymmetrik Extensible Server Framework for Healthcare allows organizations to build secure, interoperable solutions that can aggregate and expose healthcare resources via a common HL7® FHIR®-compatible REST API.
 
-The framework defines a core server, `node-fhir-server-core`, a simple, secure Node.js module built according to the FHIR specification and compliant with the [US Core](http://www.hl7.org/fhir/us/core/) implementation.
+
+The Asymmetrik Extensible Server Framework for Healthcare allows organizations to build secure, interoperable solutions that can aggregate healthcare resources via a common HL7® FHIR®-compatible REST API.
+
+The framework defines a core server, `node-fhir-server-core`, a secure Node.js module built according to the FHIR specification with the [US Core](http://www.hl7.org/fhir/us/core/) implementation.
+
 
 
 <img src="https://www.asymmetrik.com/wp-content/uploads/2018/01/FHIR-Server-Architecture_Update.png" width="800">
 
 
 ## Prerequisites
-[Node.js](https://nodejs.org/en/) version later than 7.6, **but** not 8.5 (see [Attention](#attention)), and an understanding of [promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) usage are required.
 
-Familiarity of the FHIR specification and whatever database containing the resources will assist in querying data and returning them in a format conforming to specification.
+1. Install [Node.js](https://nodejs.org/en/) latest version (7.7+, **but** not 8.5, see [Attention](#attention)) and NPM included with Nodejs.
+2. Optional: Install [Yarn](https://yarnpkg.com/lang/en/docs/install) latest version.
+
+Understanding of JavaScript [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) and familiarity of the FHIR specification would be helpful.
 
 ## Getting Started
-Install `node-fhir-server-core` as followed:
 
-```shell
-# For yarn users
-yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
+1. Install `node-fhir-server-core` initial modules with either NPM or Yarn:
 
-# For npm users
+```shell
 npm install --save git+https://github.com/Asymmetrik/node-fhir-server-core.git
+#OR
+yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
 ```
 
-Once installed, the server can be required in the application.
 
+
+2. File `node-fhir-server-core/src/runner.js` allows custom authorization server which should be incorporated separately. The server portion can be included as followed:
```

Comment:
We probably shouldn't refer them to runner.js, that is not meant for anyone to use except maintainers of this library. It is just there so we can actually test what we are building. Also, mentioning they can add custom authorization but not linking any other documentation for how to do that is probably going to lead to more confusion.

Reply:
Right. Not sure if others would know how to create their own runner / include server portion separately without looking at runner.js.

Reply:
We should link them to the mongo repo then, runner.js is not going to be setup for production and should not be used as a reference, it is just for running a dev server for us. We should also show them how to do these kinds of things in the wiki. Much rather have people read documentation versus read code.

Reply:
Ok